### PR TITLE
Update resources for oauth-proxy container

### PR DESCRIPTION
**Pull request by VaishnaviHire**
Closes: #942

This PR updates the resource limit to `oauth-proxy` container in dashboard deployment.

This is required to allow large number of concurrent requests during scale testing.

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
@andrewballantyne @LaVLaS Can you review this? Based on the discussions from today's meeting, updating resources to reduce delays when a large number of Notebooks are created.

File: manifests/base/deployment.yaml
Comment (andrewballantyne) [inline manifests/base/deployment.yaml:103]: You're requesting 1000 but limits are 500? I think you got these backwards @VaishnaviHire

**Review by andrewballantyne — CHANGES_REQUESTED** [review]

File: manifests/base/deployment.yaml
Comment (VaishnaviHire) [reply manifests/base/deployment.yaml:100]: Yes I did. Thanks!! Updated now

**Review by andrewballantyne — APPROVED** [review]
/hold

lgtm -- can merge tomorrow or if Landon gets to it (feel free to remove the hold @LaVLaS if you do)

**Pull request merged**
